CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+**0.5.0**
+
+* This release includes version **0.5.0** (exactly **0.5.0-0.5.0**) of the [zkVerify](https://github.com/HorizenLabs/zkVerify) project.
```

Comment:
If it makes sense I would add couple more words regarding why we started to use new tagging policy or what does it mean now :shrug:

Reply:
Tried to give few more details, please double check if it's fine.